circle.yml

```diff
@@ -15,6 +15,9 @@ dependencies:
     - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
     - sudo apt-get update && sudo apt-get install yarn
     - echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+    # Make sure there is no cache
+    - npm cache clean
+    - yarn cache clean
```

Comment:
Do we know why the issue is happening ? https://circleci.com/docs/1.0/how-cache-works/#clearing-cache

> The Rebuild without cache button in the UI will disable the cache for a single build so you can debug any problems caused by your dependencies.

> If that rebuild successfully passes the dependency phase, it will save a new cache which will be used by future builds.

> You shouldn’t generally need to clear the cache permanently, but if you’d like to do so, you can just remove the necessary parts of the cache anywhere in your circle.yml’s dependencies section, before the cache is saved.

It seems to me that we will be losing caching advantages (build faster) when the cache should already been cleaned up so I am not sure why we still have problems with gitbook.